content/docs/use-cases/experiment-tracking.md

```diff
@@ -0,0 +1,113 @@
+# Machine Learning Experiment Tracking
+
+Making progress on data science projects requires a large number of
+<abbr>experiments</abbr> -- attempts at tuning parameters, trying different
+data, improving code, collecting new metrics, etc. Keeping track of all these
+changes is essential, as we may need to review, compare, and share them; and to
+recover the conditions that produced certain results before in order to
+reproduce or update them.
+
+Tracking experiments traditionally meant logging scalar values with generic
+tools such as notebooks and spreadsheets. But as experiments evolve and many
+more are recorded, they can easily become disconnected and impossible to go back
+to.
+
+![](/img/natural-experimentation.png) _Loose experimentation_
+
+DVC provides a layer of [experiment management] features out-of-the-box (no need
+for special services, servers, or websites). Running **DVC Experiments** in your
+workspace captures relevant changesets automatically (input data, code,
+hyperparameters, artifacts, etc.), which can be reviewed and restored later.
+This is possible because your entire ML pipeline is [codified] with DVC.
+
+Other tools (see a
+[comparison](#comparison-of-popular-experiment-tracking-tools) below) tend to
+focus on experiment navigation, mainly logging metrics and artifacts that result
+from your pipelines, along with fragile links to code revisions. DVC's approach
+guarantees reproducibility by working on top of Git instead of as a parallel
+system.
+
+[experiment management]: /doc/user-guide/experiment-management
+[codified]: /doc/user-guide/project-structure/pipelines-files
+[versioning everything]: /doc/use-cases/versioning-data-and-model-files
+
+```dvctable
+┏━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━━━━━┓
+┃ white:**Experiment**              ┃ yellow:**loss**    ┃ yellow:**acc**    ┃ blue:**epochs** ┃ blue:**model.conv_u** ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━╇━━━━━━━━╇━━━━━━━━━━━━━━┩
+│ workspace               │ 0.23508 │ 0.9151 │ 10     │ 24           │
+│ main                    │       - │      - │ 10     │ 16           │
+│ ├── e2647ef [exp-ee8a4] │ 0.23146 │ 0.9145 │ 10     │ 64           │
+│ ├── 15c9451 [exp-a9be6] │ 0.25231 │ 0.9102 │ 10     │ 32           │
+│ └── 9c32227 [exp-17dd9] │ 0.23687 │ 0.9167 │ 10     │ 256          │
+│ 7317bc6                 │       - │      - │ 100    │ 16           │
+│ ├── 299dc5e [monograms] │ 0.24838 │ 0.9176 │ 100    │ 128          │
+│ └── 8a9cb15 [bigrams]   │ 0.24459 │ 0.9134 │ 100    │ 64           │
+└─────────────────────────┴─────────┴────────┴────────┴──────────────┘
+```
+
+![]() _DVC Experiments are organized along project versions._
```

Comment:
> Hi @rogermparent / @julieg18 when I go in https://dvc-org-cases-exp-mgmt-eur801x.herokuapp.com/doc/use-cases/experiment-tracking and click the only TOC entry (right sidebar) it doesn't scroll to anchor #comparison-of-popular-experiment-tracking-tools. But it works if I disable Javascript... Ideas? (Please start a separate issue from this comment if needed). Thanks

This syntax creates an broken `<img/>` element. Our scrolling functions wait till all images are loaded then scroll to desired element. Since this broken img element never loads, the page never scrolls to desired position. 

Hello! We could either update our scrolling function to check for broken images and ignore them or we could take off the empty image. Personally, I think we should take off the `![]()`, since adding empty images to a page is bad practice.

Reply:
Great catch! Agreed we shouldn't be using empty images, maybe we should add support/styling to the `figure`/`figcaption` elements or some sort of custom component that emulates what this is doing- the ability to caption code blocks does seem valuable.

Reply:
Ah, that makes sense. It'd be nice both to fix the engine and especially to have a special `<figcaption>` tag.

For now I tried copying the HTML from the previous fig. capture, but it throws an error:

<details>

```html
<span class="gatsby-resp-image-wrapper" style="position: relative; display: block; margin-left: auto; margin-right: auto; max-width: 700px;">
  <em>
    _DVC Experiments are organized along project versions._
  </em>
</span>
```

![image](https://user-images.githubusercontent.com/1477535/144496680-9ef8dde6-b342-4bf9-b7ef-f4315cd79ac0.png)

It works if I remove the `class` but it doesn't look like a caption.

</details>

Reply:
> p.s. never mind I copied the entire styles now (instead of class) and it works. See 2f6416fb.